node/tests/SharedTests.ts

```diff
@@ -10637,8 +10637,8 @@ export function runBaseTests(config: {
                 expect(result[0].pending).toEqual(1);
                 expect(result[0].idle).toBeGreaterThan(0);
 
-                if (!cluster.checkIfServerVersionLessThan("7.2.0")) {
-                    expect(result[0].inactive).toBeGreaterThan(0);
+                if (cluster.checkIfServerVersionLessThan("7.2.0")) {
+                    expect(result[0].idle).toBeGreaterThan(0); // Use `idle` for Redis versions before 7.2.0
```

Comment:
`idle` checked to lines above

Reply:
@Yury-Fridlyand  Deleted the WAIT transactions test completley.
I've seen in your PR you're running this test for versions greater than 7.0.0. but note that it's also failing for 7.2
the whole WAIT in the transactions is flaky as it seems to be relevant to how fast the engine handles it, sometimes it does succeed on 6.2 and sometimes it fails, so we can't be completely sure how it behaves
Anyhow, we agreed upon deleting the test and merging this PR since we need it ASAP